Dockerfile

```diff
@@ -1,19 +1,17 @@
 FROM golang:1.7-alpine
 
 WORKDIR /go/src/app
-ENV ALLOW_CONTAINER_ROOT=1
 
 COPY . /go/src/app
 RUN \
 	apk add --no-cache git && \
 	go-wrapper download && \
-	go-wrapper install && \
+	go-wrapper install -ldflags "$(go run buildscripts/gen-ldflags.go)" && \
 	mkdir -p /export/docker && \
 	cp /go/src/app/docs/Docker.md /export/docker/ && \
 	rm -rf /go/pkg /go/src && \
 	apk del git
 
 EXPOSE 9000
-ENTRYPOINT ["go-wrapper", "run", "server"]
+ENTRYPOINT ["go-wrapper", "run"]
```

Comment:
We had to make this breaking change since minio has other sub-commands, it was our mistake that "server" was included as part of the entrypoint earlier. 

To run docker properly now the right command is 

```
docker run -p 9000:9000 minio/minio server /export
```